src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -139,41 +137,35 @@ private Future<Void> processInventory(CompositePoLine compPOL, boolean deleteHol
     }
     if (PoLineCommonUtil.isInventoryUpdateNotRequired(compPOL) || PoLineCommonUtil.isOnlyInstanceUpdateRequired(compPOL)) {
       return deleteExpectedPieces(compPOL, requestContext).onSuccess(pieces -> {
-        if (logger.isDebugEnabled()) {
-          String deletedIds = pieces.stream().map(Piece::getId).collect(Collectors.joining(","));
-          logger.debug(String.format("Pieces were removed : %s", deletedIds));
-        }
-      })
-      .mapEmpty();
+          if (logger.isDebugEnabled()) {
```

Comment:
seems auto-formatting is not correct here, because we have 4 spaces instead of 2 now